experimental/GITFans/GITFans.jl

```diff
@@ -336,7 +335,7 @@ function compute_bit_list(orbits, point)
         current_orbit = orbits[current_orbit_nr]
         for current_cone_nr in 1:length(current_orbit)
             current_cone = current_orbit[current_cone_nr]
-            if Polymake.polytope.contains(current_cone, point)
+            if contains(current_cone, point)
```

Comment:
Dunno if this is supported, but if it is, IMHO it would be easier to read
```suggestion
            if point in current_cone
```

Reply:
I added that to our todolist, but I think it makes sense to do this in a different PR.